manifests/p/PixPin/PixPin/1.0.9.1/PixPin.PixPin.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: PixPin.PixPin
+PackageVersion: 1.0.9.1
+InstallerType: inno
+ProductCode: "{506270E7-E3B5-4157-B53A-3BFFE8E418DB}_is1"
+Installers:
+- InstallerUrl: https://download.pixpinapp.com/PixPin_1.0.9.1.exe
+  Architecture: x86
```

```suggestion
  Architecture: x64
```

According to [this feedback](https://txc.qq.com/embed/phone/614512/post/170194352455462128/), it seems this package is a 64-bit application.